app/controllers/ApplicationController.scala

```diff
@@ -296,6 +296,36 @@ trait ApplicationController extends BaseController with GatekeeperAuthWrapper {
     }
   }
 
+  def unblockApplicationPage(appId: String) = requiresRole(Role.APIGatekeeper, requiresSuperUser = true) {
+    implicit request => implicit hc => withApp(appId) { app =>
+      Future.successful(Ok(unblock_application(app, isSuperUser, unblockApplicationForm.fill(UnblockApplicationForm("")))))
+    }
+  }
+
+  def unblockApplicationAction(appId: String) = requiresRole(Role.APIGatekeeper, requiresSuperUser = true) {
+    implicit request => implicit hc => withApp(appId) { app =>
+      def handleValidForm(form: UnblockApplicationForm) = {
+        if (app.application.name == form.applicationNameConfirmation) {
+          applicationService.unblockApplication(appId, loggedIn.get).map {
+            case ApplicationUnblockSuccessResult => Ok(unblock_application_success(app, isSuperUser))
```

Comment:
Similarly should this have a issuperuser param as well as needing to be a superuser to enter this method?